tests/sentry/integrations/slack/test_utils.py

```diff
@@ -103,3 +103,193 @@ def test_rate_limiting(self):
         )
         with pytest.raises(ApiRateLimitedError):
             get_channel_id(self.organization, self.integration, "@user")
+
+
+class GetChannelIdFasterTest(TestCase):
+    @with_feature("organizations:slack-use-new-lookup")
+    def setUp(self):
+        self.resp = responses.mock
+        self.resp.__enter__()
+
+        self.integration = install_slack(self.event.project.organization)
+        self.add_list_response(
```

Comment:
I don't think you need this since we're not hitting the `conversations.list` endpoint.